docs/Displaying a User Interface/Template Buttons/index.md

```diff
@@ -0,0 +1,561 @@
+    @Override
+    public void onPress(SoftButtonObject softButtonObject, OnButtonPress onButtonPress) { 
+    }
+
+    @Override
+    public void onEvent(SoftButtonObject softButtonObject, OnButtonEvent onButtonEvent) {
+    }
+});
+
+sdlManager.getScreenManager().beginTransaction();
+sdlManager.getScreenManager().setSoftButtonObjects(Collections.singletonList(softButtonObject));
+sdlManager.getScreenManager().commit(new CompletionListener() {
+	@Override
+	public void onComplete(boolean success) {
+		Log.i(TAG, "ScreenManager update complete: " + success);
+	}
+});
+
+// Transition to a new state
+SoftButtonObject retrievedSoftButtonObject = sdlManager.getScreenManager().getSoftButtonObjectByName("softButtonObject");
+retrievedSoftButtonObject.transitionToNextState();
+```
+!@
+
+### Deleting Soft Buttons
+To delete soft buttons, simply pass the screen manager a new array of soft buttons. To delete all soft buttons, simply pass the screen manager an empty array.
+
+@![iOS]
+##### Objective-C
+```objc
+self.sdlManager.screenManager.softButtonObjects = @[];
+```
+
+##### Swift
+```swift
+sdlManager.screenManager.softButtonObjects = []
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+sdlManager.getScreenManager().setSoftButtonObjects(Collections.EMPTY_LIST);
+```
+!@
+
+## Subscribing to System Buttons
+Subscribe buttons are used to detect when your user has interacted with buttons located in the car's center console or steering wheel. You can subscribe to the following buttons:
+
+| Button  | Template |
+| ------------- | ------------- |
+| Play / Pause | Media only |
+| Ok | Media only |
+| Seek left | Media only |
+| Seek right | Media only |
+| Tune up | Media only |
+| Tune down | Media only |
+| Preset 0-9 | Any |
+| Search | Any |
+| Center Location | NavigationFullscreenMap only |
+| Zoom In | NavigationFullscreenMap only |
+| Zoom Out | NavigationFullscreenMap only |
+| Pan Up | NavigationFullscreenMap only |
+| Pan Up-Right | NavigationFullscreenMap only |
+| Pan Right | NavigationFullscreenMap only |
+| Pan Down-Right | NavigationFullscreenMap only |
+| Pan Down | NavigationFullscreenMap only |
+| Pan Down-Left | NavigationFullscreenMap only |
+| Pan Left | NavigationFullscreenMap only |
+| Pan Up-Left | NavigationFullscreenMap only |
+| Toggle Tilt | NavigationFullscreenMap only |
+| Rotate Clockwise | NavigationFullscreenMap only |
+| Rotate Counter-Clockwise | NavigationFullscreenMap only |
+| Toggle Heading | NavigationFullscreenMap only |
+
+!!! NOTE
+`Media` is the default template for media apps, and `NavigationFullscreenMap` is the default template for navigation apps.
+!!!
+
+### Subscribe Buttons HMI
+In the screenshot below, the pause, seek left and seek right icons are subscribe buttons.
+![Generic - Media Template with subscribe buttons](assets/Generic_template_media_light.png)
+
+!!! NOTE
+There is no way to customize a subscribe button's image or text.
```

Comment:
This isn't totally true for the play/pause button, which you can (limitedly) change using `SDLSetMediaClockTimer.audioStreamingIndicator`

Reply:
I removed this note and added a paragraph with a link to **Media Clock** section.